### ISSUE TITLE
Can you add this module like the official chatgpt does?

### ISSUE THREAD

**Issue by Root-FTW** [ddiu8081/chatgpt-demo#150]
### Clear and concise description of the problem

![2023_03_06_13_31_20](https://user-images.githubusercontent.com/108767514/223236949-73bb4476-1a69-4858-8896-adcb862995a8.png)

### Suggested solution

Can you add this module like the official chatgpt does?

It would be great to have 3 columns of 3 items each of promts examples.

And that those promts/examples can be changed/managed from the source code.

Thanks

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ddiu8081/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/ddiu8081/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by ddiu8081** [comment]
I like it, thanks for your idea!

**Comment by Root-FTW** [comment]
> I like it, thanks for your idea!

Thanks to you for such a beautiful project, when will you have your kofi page to make donations? you deserve it <3

**Comment by luoxiaofeng1029** [comment]
> 


I also strongly agree. I have never loved a project so much. I think your hard work deserves a cup of coffee from the people who use your project. Please add a coffee channel for us to show our appreciation.

**Comment by mrslimslim** [comment]
I agree,and I think this will be a trend to provide high quality promts in different fields.It deserve thing deeper not just copy.
